ice/recipes/consort_flow/baselines.py

```diff
@@ -349,6 +369,27 @@ async def _all_options(
     )
 
 
+async def _cheating_elicit_qa_baseline(
+    paper: Paper,
+    question: str,
+    gold_support: Sequence[str] | None,
+):
+    if gold_support is None:
+        gold_support = []
+    gold_support_strs = await identify_gs_str(_to_paragraphs(paper), gold_support)
```

Comment:
this uses the paragraph from the paper that includes the gold standards, rather than just the gold standard itself

this is a more-relevant baseline for us, and it matches the cheating baselines for the other recipes e.g. recipe 6

Reply:
@CG80499